### ISSUE TITLE
Add k-p-t  tests?

### ISSUE THREAD

**Issue by saaratrix** [saaratrix/glossarytraining#19]
Add some kind of test for the grammar when bending words?
```
vahva  - heikko 
kk -> k
tt -> t
pp -> p

k -> -
p -> v
t -> d

...
```

**Comment by saaratrix** [comment]
Will add a new grammar test that takes 0 -> n phrases and then a hard coded bending for that phrase. So it's very universal and can be used for almost anything.

**Comment by saaratrix** [comment]
Closed by PR #32
